config/rbac/submariner-globalnet/cluster_role.yaml

```diff
@@ -26,6 +26,17 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - submariner.io
+    resources:
+      - clusterglobalegressips
+      - globalegressips
+      - globalingressips
+    verbs:
+      - get
+      - list
+      - watch
+      - update
```

Comment:
I guess we can update the perms for pods etc (L11) above later.

Reply:
that's correct. We can update them after we get the basic connectivity working with the new Globalnet 2.0 code.